src/containers/RepositoriesContainer/RepositoriesDucks.js

```diff
@@ -0,0 +1,37 @@
+// Actions
+const LOAD = 'react-redux-starter-app/repositories/LOAD';
+const LOAD_SUCCESS = 'react-redux-starter-app/repositories/LOAD_SUCCESS';
+const LOAD_FAIL = 'react-redux-starter-app/repositories/LOAD_FAIL';
+
+const initialState = {
+  isLoading: false,
+  repos: undefined
```

Comment:
I was going to differentiate between unloaded (i.e. `undefined`) vs loaded but no repos (i.e.`[]`)